### PR TITLE
fix(ui): removed project name from file source breadcrumbs (#1326)

### LINKED ISSUE #1326: Remove project name before file name
The project name is showing up in front of every file. That's not necessary.

image

### PR DESCRIPTION
**Pull request by finleygn**
Fixes #1326

image

### CODE REVIEW

**Review by xla — CHANGES_REQUESTED** [review]
Thanks a lot for contributing. Plese see https://github.com/radicle-dev/radicle-upstream/pull/1467#pullrequestreview-546051287

**Review by cloudhead — DISMISSED** [review]
Thanks for contributing!

File: ui/DesignSystem/Component/SourceBrowser/FileView/Blob.svelte [hunk 1]
```diff
@@ -3,13 +3,11 @@
...
 
-  export let rootName: string;
   export let view: Blob;
 </script>
 
 <FileSource
   blob={view.blob}
   commit={view.blob.info.lastCommit}
-  {rootName}
   on:root
```
Comment (xla) [inline ui/DesignSystem/Component/SourceBrowser/FileView/Blob.svelte:12]:
```suggestion
```

FileSource doesn't expose this anymore so we can remove it.

File: ui/DesignSystem/Component/SourceBrowser/FileView.svelte [hunk 2]
```diff
@@ -24,7 +23,7 @@
 
 <div data-cy="file-view">
   {#if view.kind === ViewKind.Blob}
-    <Blob {rootName} {view} on:root={onSelectRoot} on:select={onSelectCommit} />
+    <Blob {view} on:root={onSelectRoot} on:select={onSelectCommit} />
```
Comment (xla) [inline ui/DesignSystem/Component/SourceBrowser/FileView.svelte:26]:
```suggestion
    <Blob {view} on:select={onSelectCommit} />
```

The `on:root` binding is not necessary anymore as the origin event was removed.

**Review by xla — CHANGES_REQUESTED** [review]
Great work, just a small clean up we can do.

File: ui/DesignSystem/Component/SourceBrowser/FileSource.svelte [hunk 3]
```diff
@@ -89,11 +75,7 @@
     <div class="file-header typo-semi-bold" data-cy="file-header">
       <Icon.File />
       <span class="file-name">
-        <span
-          class="typo-link root-link"
-          data-cy="root-link"
-          on:click={() => dispatch('root')}>{rootName}</span>
-        <span>/ {blob.path.split('/').join(' / ')}</span>
+        <span>{blob.path.split('/').join(' / ')}</span>
       </span>
```
Comment (MeBrei) [inline ui/DesignSystem/Component/SourceBrowser/FileSource.svelte:79]: The double nested `span` could now be removed.

**Review by MeBrei — APPROVED** [review]
Thanks for the contribution! ⭐

**Comment by finleygn** [comment]
all updated :)

**Review by MeBrei — APPROVED** [review]

**Review by xla — APPROVED** [review]
🌻 ✳️ 🍜 🏆

**Review by rudolfs — CHANGES_REQUESTED** [review]
I ran the specs locally with `yarn test` (as a workaround for https://github.com/radicle-dev/radicle-upstream/issues/1471), and three of them need to be updated before we can merge this:

```
  3 failing

  1) project source browsing
       source code browsing
         page view
           when we're looking at the project root
             when there is a README file
               shows the README file:
     AssertionError: Timed out retrying: Expected to find content: 'platinum / .i-am-well-hidden' within the element: <div> but never did.
      at Context.eval (http://localhost:64825/__cypress/tests?p=cypress/integration/project/source_browsing.spec.ts:242:30)

  2) project source browsing
       source code browsing
         source-tree
           doesn't interfere with the horizontal menu item active state:
     AssertionError: Timed out retrying: Expected to find content: 'platinum' within the element: <div.file-header.typo-semi-bold.svelte-1ub4xlw> but never did.
      at Context.eval (http://localhost:64825/__cypress/tests?p=cypress/integration/project/source_browsing.spec.ts:358:100)

  3) project source browsing
       source code browsing
         source-tree
           when clicking on a file name
             for filenames with special characters
               does not break:
     AssertionError: Timed out retrying: Expected to find content: 'platinum / special / -dash-' but never did.
      at Context.eval (http://localhost:64825/__cypress/tests?p=cypress/integration/project/source_browsing.spec.ts:431:68)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        24                                                                               │
  │ Passing:      19                                                                               │
  │ Failing:      3                                                                                │
  │ Pending:      2                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  3                                                                                │
  │ Video:        false                                                                            │
  │ Duration:     46 seconds                                                                       │
  │ Spec Ran:     project/source_browsing.spec.ts                                                  │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

**Comment by juliendonck** [comment]
Work is moved to #1609

**Pull request closed without merging**